src/goose.rs

```diff
@@ -1529,16 +1533,16 @@ impl GooseUser {
 
         // Send raw request object to parent if we're tracking statistics.
         if !self.config.no_stats {
-            self.send_to_parent(&raw_request)?;
+            self.send_request_stats_to_parent(&raw_request)?;
```

Comment:
As spoken: `self.send_to_parent` is better as there is no other function, so there is no ambiguity,

Reply:
restored